packages/mui-icons-material/package.json

```diff
@@ -83,5 +83,22 @@
   },
   "engines": {
     "node": ">=14.0.0"
+  },
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "import": "./esm/index.js",
+      "require": "./index.js"
+    },
+    "./*": {
+      "types": "./*.d.ts",
+      "import": "./esm/*.js",
+      "require": "./*.js"
+    },
+    "./esm/*": {
```

Comment:
Maintain backwards compatibility for users that have done https://github.com/mui/material-ui/blob/8b8732eabd272226fdc858cdfc1dce8b0ce4b294/apps/pigment-css-vite-app/vite.config.ts#L48-L55